test/commandLine/app.txt

```diff
@@ -0,0 +1,10 @@
+const path = require('path')
```

Comment:
rename this to `app.js`, move it to the `test/lib` directory, and ensure that the code indentation is 2 spaces and not 4.